website/docs/plugin/framework/migrating/providers/index.mdx

```diff
@@ -63,6 +63,28 @@ configuration. Use muxing when you want to release a version of your provider wi
 sources migrated to the Framework. You may want to do this if your provider manages a large number of resources and
 data sources.
 
+
+~> **Important**: If you are using muxing and your SDK provider schema is using
```

Comment:
Doesn't have to be changed now, but FYI about new callout syntax: https://github.com/hashicorp/terraform-providers-devex-internal/issues/124

Reply:
Will come back to this one. Using <Highlight>....</Highlight> resulted in backtick highlighting being ignored and output literally.